chain/chain/src/chain.rs

```diff
@@ -2481,6 +2514,8 @@ pub struct ChainUpdate<'a> {
     genesis: &'a Block,
     #[allow(unused)]
     transaction_validity_period: BlockHeightDelta,
+    #[cfg(feature = "sandbox")]
```

Comment:
Guess general consensus was that we'll try to avoid Rust features whenever possible.
cc: @matklad

Reply:
I removed most of them on https://github.com/near/nearcore/pull/4287/commits/81fa0e065cde46ff941bdf0b5575e54080d6e2f4, but this place I think it's better to have sandbox feature. chain need a sandbox feature because Chain is the place of adding sandbox related features, Chain needs to know whether sandbox feature present and implement logic only when sandbox present, so it makes sense some fields of Chain is only available under sandbox.

Reply:
As it's not core of this change, I'm OK putting feature for now, although still believe that in longer run it doesn't make sense to use features.